<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BidResultWidget</class>
 <widget class="QWidget" name="BidResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="BidResultFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>266</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#BidResultFrame{border-image: url(:/image/bidResultWidget/loading.png);}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>205</y>
      <width>542</width>
      <height>8</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
border-radius: 5px;background:#551053;
}

QProgressBar::chunk {
border-radius:5px;background:#fbd12b;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image: url(:/image/bidResultWidget/bt_close_hover.png);}
QPushButton{border-image: url(:/image/bidResultWidget/bt_close.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/image/bidResultWidget/bt_min.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>8</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;span style=&quot; font-family:宋体; font-size:14px; color:#ffffff;&quot;&gt;&lt;center&gt;&lt;b&gt;斗歌正在加载&lt;/b&gt;&lt;/center&gt;&lt;/span&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="processBarLabel">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>191</y>
      <width>38</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/bidResultWidget/progress_bar_anniu.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tipsLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:75 &quot;宋体&quot; 12px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>唱歌当然要准备好耳麦啦～</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>201</y>
      <width>552</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/bidResultWidget/progress_bar_2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>progressBar</zorder>
   <zorder>closeBtn</zorder>
   <zorder>minBtn</zorder>
   <zorder>titleLabel</zorder>
   <zorder>processBarLabel</zorder>
   <zorder>tipsLabel</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
